week1/homework/package.json

```diff
@@ -7,7 +7,8 @@
   "author": "George Sapkin",
   "license": "MIT",
   "scripts": {
-    "test": "ava --verbose",
+    "start": "node src/index.js",
```

Comment:
Have you run npm init in the homework folder?
It already had a package .json so you did not need to run  it

Reply:
WIll have to ask you about this again as I don´t seem to understand the question just yet. Did you mean because of the "test" part?
I have been running npm start in the terminal

Reply:
Yes, how come the test part has been removed?